scripts/errorCheck.mjs

```diff
@@ -27,7 +27,7 @@ async function checkErrorBaselines() {
             let g;
             while (g = errRegex.exec(baseline)) {
                 const errCode = +g[1];
-                const msg = keys.filter(k => messages[k].code === errCode)[0];

```

Comment:
This just seems like a weird pattern in general.  Particularly since `filter` doesn't short-circuit.

Reply:
I don't disagree, I didn't write it, but I am fixing it 😅